docs/tutorials/configure/build-a-mock-robot.md

```diff
@@ -20,50 +20,67 @@ date: "2022-10-11"
 
-## Requirements
+By utilizing {{< glossary_tooltip term_id="part" text="parts" >}}, you can expand upon this, chaining multiple computers together to build your robot with Viam:
+
+- Each individual computer-controlled unit of a robot is called a “{{< glossary_tooltip term_id="part" text="part" >}}” in Viam.
+- Typically, simple robots have just one part, but you can have as many parts as your project requires.
+- Parts are organized in a tree, with one of them being the _main_ part, and the others being _sub-parts_.
+- You can access any sub-part either directly, or through any part above it in the tree.
+- Each part runs a single `viam-server` instance.
+
+### What is a `fake` component?
+
+Most Viam [components](../../../components/) have a `fake` model that you can use for testing.
+These `fake` components interact with Viam like real hardware but do not actually exist.
+
+## Set up a mock robot
 
 You'll need the following hardware and software for this tutorial:
 
-- A laptop or desktop running Linux or macOS
+- A computer running Linux or macOS
 - [Go](https://go.dev/doc/install) or [Python 3.9+](https://www.python.org/downloads/)
-- A code editor of your choice
 
-## Set up a mock robot
+If you don't already have an account, create a free one.
+Sign up on [the Viam app](https://app.viam.com).
+Create a new robot in the organization and location of your choice.
+Go to this robot's **Setup** tab.
 
-### Make an account on the Viam app
+### Install and start `viam-server` on your computer
 
-The first thing you need to do is set up your account on the Viam app.
-Go to [app.viam.com](https://app.viam.com) and sign up for a free account.
+Before you proceed with configuring and controlling your robot, install `viam-server`.
+Follow the steps outlined for your computer's architecture on the **Setup** tab of the [Viam app](https://app.viam.com) to install `viam-server` on your computer as a system service.
+Find more information on running `viam-server` in the [installation guide](/installation/manage/).
 
 ### Configure your mock robot
 
-Now you'll [configure your robot](/manage/configuration/) to represent your robot's hardware.
-If you were using actual hardware, this file would tell your code what hardware is attached to it and how to communicate with it.
+[Configure your mock robot](/manage/configuration/) to represent a physical smart machine with robotic board, arm, and motor hardware.
 
-Since this is an imaginary robot, you will use `fake` components so that the Viam software ([`viam-server`](../../../viam/#get-started)) doesn't try and fail to connect to physical hardware.
+If you were using actual hardware, this process would provide `viam-server` with the information what hardware is attached to it and how to communicate with it.
```

Comment:
```suggestion
If you were using physical hardware, this process would provide `viam-server` with the information what hardware is attached to it and how to communicate with it.
```
Just feel like "actual" sounds a bit judgmental - might be overreacting

Reply:
Fine with me!